rest/api.go

```diff
@@ -274,8 +274,13 @@ func (h *handler) handleGetResync() error {
 }
 
 func (h *handler) handlePostResync() error {
+
 	action := h.getQuery("action")
 	regenerateSequences, _ := h.getOptBoolQuery("regenerate_sequences", false)
+	body, err := h.readBody()
```

Comment:
I'd prefer to do all the payload parsing/query parameter handling here in rest, instead of passing the full payload body through as 'collections' to ResyncManager.Start(), to keep all the REST API-specific handling in the rest package.